dot/network/service.go

```diff
@@ -430,6 +430,25 @@ func (s *Service) sentBlockIntervalTelemetry() {
 func (s *Service) handleConn(conn libp2pnetwork.Conn) {
 	// TODO: currently we only have one set so setID is 0, change this once we have more set in peerSet.
 	s.host.cm.peerSetHandler.Incoming(0, conn.RemotePeer())
+
+	// exchange BlockAnnounceHandshake with peer so we can start to
+	// sync if necessary.
+	prtl, has := s.notificationsProtocols[BlockAnnounceMsgType]
```

Comment:
Why not `protocol` :cry:

Reply:
there's a package already imported at `protocol`, i can change it though if you want!

Reply:
Ah yes, one of the dumb Go feature/bug 😢 

Maybe have it as `proto`? Or rename the import to `protocolpkg` 🤔 Up to you!